### PR TITLE
Alpha/dialog-update

### PR DESCRIPTION
**Pull request by iondrimba**
**CHANGELOG** :memo:

* Updated Dialog component
* Added Dialog inside-wall modifier
* Added Dialog left modifier
* Added Dialog centered modifier
* Updated documentation

screenshot capture - 2017-04-28 - 17-46-33

screenshot capture - 2017-04-28 - 17-46-27

captura de tela 2017-04-27 as 6 25 12 pm

### CODE REVIEW

File: docs/demo/gaiden-css/components/dialog/demo.html [hunk 1]
```diff
@@ -9,15 +9,15 @@
...
     <div class="dialog col-small-12 col-normal-5 space-box-large">
       <blockquote class="dialog__quote">
-        <p>Precisava fazer um telhado no meu quintal, entrei no GetNinjas e fiz um pedido de pedreiro, eles entraram em contato comigo no mesmo dia, escolhi o que melhor me atendeu e fechei o serviço.</p>
+        <p>"Precisava fazer um telhado no meu quintal, entrei no GetNinjas e fiz um pedido de pedreiro, eles entraram em contato comigo no mesmo dia, escolhi o que melhor me atendeu e fechei o serviço."</p>
       </blockquote>
 
       <div class="dialog__user space-box-medium">
         <picture class="dialog__user-photo picture picture--rounded">
-          <img src="http://placehold.it/40x40" alt="Avatar for Silvia D." class="picture__image" height="40" width="40">
+          <img src="http://placehold.it/40x40" alt="Avatar for Silvia D." class="picture__image" height="30" width="30">
```
Comment (eduardojmatos) [inline docs/demo/gaiden-css/components/dialog/demo.html]: http://placehold.it/30x30, like another examples above 😄

File: src/scss/components/_dialog.scss [hunk 2]
```diff
@@ -47,12 +55,35 @@
...
-      padding-top: 4px;
     }
 
     &-location {
       font-size: 12px;
       line-height: 15px;
     }
   }
+
+  &--inverted {
```
Comment (eduardojmatos) [inline src/scss/components/_dialog.scss]: @iondrimba `--inside-wall` could be more explicit here instead `inverted`. Wall is a container component that have a background-color rastaGray. What do you think?

Comment (iondrimba) [inline src/scss/components/_dialog.scss]: @eduardojmatos Perfect!

File: docs/demo/gaiden-css/components/dialog/demo.html (hunk 1 above)
Comment (iondrimba) [inline docs/demo/gaiden-css/components/dialog/demo.html]: @eduardojmatos will change it

**Comment by iondrimba** [comment]
@eduardojmatos Suggested changes applied :+1:

File: docs/demo/gaiden-css/components/dialog/aligned-left/demo.html [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+  <link rel="stylesheet" href="/gaiden-css/base.css">
+</head>
+<body>
+  <section class="container wall content-centered col-centered">
+    <div class="dialog dialog--aligned-left col-small-12 col-normal-5 space-box-large">
+      <blockquote class="dialog__quote">
+        <p>"Precisava fazer um telhado no meu quintal, entrei no GetNinjas e fiz um pedido de pedreiro, eles entraram em contato comigo no mesmo dia, escolhi o que melhor me atendeu e fechei o serviço."</p>
+      </blockquote>
+
+      <div class="dialog__user space-box-small">
```
Comment (MarcoBrunoBR) [inline docs/demo/gaiden-css/components/dialog/aligned-left/demo.html]: @iondrimba just to have more semantics. Is it worth changing to a ```dl>(dt> picture> img)+dd>div*2```?

File: src/scss/components/_dialog.scss [hunk 4]
```diff
@@ -15,20 +21,21 @@
...
     position: relative;
-    background-color: get-color(air);
+    background-color: get-color(breeze);
     border-radius: 10px;
     align-items: center;
     justify-content: center;
     font-style: italic;
+    color: get-color(rastaGray);
 
     &::after {
```
Comment (MarcoBrunoBR) [inline src/scss/components/_dialog.scss]: @iondrimba, do you think it's worth creating a mixin?

Comment (iondrimba) [inline src/scss/components/_dialog.scss]: @MarcoBrunoBR I would if we had the same usage in other components. Now only dialogs uses these configs.

Comment (MarcoBrunoBR) [inline src/scss/components/_dialog.scss]: Nice :-)

**Comment by iondrimba** [comment]
@MarcoBrunoBR Updated like you suggested :+1:

File: docs/demo/gaiden-css/components/dialog/aligned-left/demo.html (hunk 3 above)
Comment (iondrimba) [inline docs/demo/gaiden-css/components/dialog/aligned-left/demo.html]: @MarcoBrunoBR This change will break the existing layout, I think we could talk about semantics on another PR, @eduardojmatos what do you think?

**Comment by iondrimba** [comment]
[(5) Update da versão 2 (design) do Gaiden](https://trello.com/c/k2izpoRz/406-5-update-da-versao-2-design-do-gaiden)

**Comment by iondrimba** [comment]
@eduardojmatos Added a centered modifier

File: docs/demo/gaiden-css/components/dialog/aligned-left/demo.html (hunk 3 above)
Comment (eduardojmatos) [inline docs/demo/gaiden-css/components/dialog/aligned-left/demo.html]: https://github.com/getninjas/gaiden/pull/96/files#r114005702

**Pull request merged**